### DISCUSSION TITLE
Why an extra level is included when using expand_grid and a dictionary?

### DISCUSSION THREAD

**Discussion by luifrancgom** [pyjanitor-devs/pyjanitor#1373] [Q&A] [answered]
I want to know why an extra level is included when using `expand_grid` and a dictionary?

### Minimal reproducible example

```python
data = {"x": [1, 2, 3], "y": [1, 2]}
expand_grid(others=data)
```
### Actual behavior
```
   x  y
   0  0
0  1  1
1  1  2
2  2  1
3  2  2
4  3  1
5  3  2
```
### Expected behavior

```
   x  y
0  1  1
1  1  2
2  2  1
3  2  2
4  3  1
5  3  2
```

### Settings

`pyjanitor`: 0.27
 `python`: 3.9.19

**Comment by samukweku** [comment]
hi @luifrancgom and apologies for the late reply. A PR is in the works to change this - a MultiIndex will be created only if necessary.

**Reply by luifrancgom** [reply]
No apologies. Thank you for developing this package and for working in a PR.

**Reply by samukweku** [reply, marked as answer]
this is the relevant [PR](#1383)

**Comment by luifrancgom** [comment]
Ok. Solved with #1383
